tools/dnulist/program.cs

```diff
@@ -0,0 +1,241 @@
+            }
+            else if (commands.ContainsKey(Target) && commands.ContainsKey(Copy) && !string.IsNullOrEmpty(commands[Copy]))
+            {
+                bool verbose = commands.ContainsKey(Verbose);
+                CopyTarget(project, commands[Target], commands[Copy], verbose);
+            }
+            else if (commands.ContainsKey(Target))
+            {
+                bool verbose = commands.ContainsKey(Verbose);
+                PrintTarget(project,commands[Target], verbose);
+            }
+            else if (commands.ContainsKey(Help))
+            {
+                Console.WriteLine();
+                PrintHelp();
+            }
+        }
+    }
+    
+    private static void PrintTargets(Project project)
+    {
+        Console.WriteLine ("This project has {0} targets.", project.Targets.Count);
+        Console.WriteLine();
+        
+        foreach (var t in project.Targets.Keys)
+        {
+            Console.WriteLine(t);
+        }
+    }
+    
+    private static void PrintTarget(Project project, string target, bool verbose = false)
+    {
+        Dictionary<string,Package> packages;
+        var targetFound = project.Targets.TryGetValue(target, out packages);
+        
+        if (targetFound && packages.Count > 0)
+        {
+            Console.WriteLine("Package dependencies for target: {0}",target);
+            
+            foreach (var package in packages.Keys)
+            {
+                Console.WriteLine(package);
+                if (verbose)
+                {
+                    var p = packages[package];
+               
+                    if (p.Compile != null)
+                    {
+                        foreach (var key in p.Compile.Keys)
+                        {
+                            Console.WriteLine("**Compile: {0}", key);
+                        }
+                    }
+                    
+                    if (p.Runtime != null)
+                    {
+                        foreach (var key in p.Runtime.Keys)
+                        {
+                            Console.WriteLine("**Runtime: {0}", key);
+                        }
+                    }
+                    
+                    if (p.Dependencies != null)
+                    {
+                        foreach (var key in p.Dependencies.Keys)
+                        {
+                            Console.WriteLine("**Dependencies: {0}", key);
+                        }
+                    }
+                }
+            }
+        }
+        else if (targetFound)
+        {
+            Console.WriteLine("No packages found for {0}.",target);
+        }
+        else
+        {
+            Console.WriteLine("{0} is not a target for this project.", target);
+        }
+    }    
+    
+    private static void CopyTarget(Project project, string target, string path, bool verbose = false)
+    {
+        const string packageRoot = "/Users/richlander/.dnx/packages";
```

Comment:
None of these environment variables seem to be defined under 1.0.0-beta6-12100.

Having DNX_HOME would be nice, but that shouldn't be the primary answer for this.